src/omero_zarr/raw_pixels.py

```diff
@@ -134,30 +140,20 @@ def add_raw_image(
                     plane = next(planes)
                 if plane is None:
                     continue
-                for level in range(level_count):
-                    size_y = plane.shape[0]
-                    size_x = plane.shape[1]
-                    # If on first plane, create a new group for this resolution level
-                    if len(field_groups) <= level:
-                        field_groups.append(
-                            parent.create(
-                                str(level),
-                                shape=(size_t, size_c, size_z, size_y, size_x),
-                                chunks=(1, 1, 1, size_y, size_x),
-                                dtype=d_type,
-                            )
-                        )
-
-                    # field_group = field_groups[level]
-                    field_groups[level][t, c, z, :, :] = plane
-
-                    if (level + 1) < level_count:
-                        # resize for next level...
-                        plane = cv2.resize(
-                            plane,
-                            dsize=(size_x // 2, size_y // 2),
-                            interpolation=cv2.INTER_NEAREST,
-                        )
+                z_stack.append(plane)
+            c_stack.append(np.stack(z_stack, axis=0))
+        t_stack.append(np.stack(c_stack, axis=0))
+    np_image = np.stack(t_stack, axis=0)
+
+    print("np_image", np_image.shape)
+    print("metadata", metadata)
+
+    size_x = np_image.shape[-1]
+    size_y = np_image.shape[-2]
+
+    write_image(
+        image=np_image, group=parent, chunks=(1, 1, 1, size_y, size_x), omero=metadata
```

Comment:
The only thing that this makes me aware of is that this might not fit in memory, so a generator strategy might be needed.